### PR TITLE
DM-47893: Add TAI time difference

### PR DESCRIPTION
**Pull request by ugyballoons**
Also bug fixes for:
Night report link not showing up
Faulty table controls logic

### CODE REVIEW

File: src/js/components/TableApp.js [hunk 1]
```diff
@@ -138,19 +137,26 @@ TableApp.propTypes = {
...
+    new Set(defaultColNames.concat(allColNames))
+  )
   // filter out the indicators (first char is '_')
   // and the replacement strings for empty channels
   // (first char is '@')
-  return allCols.filter((el) => el[0] !== "_" || el[0] !== "@")
+  const filtered = uniqueColNames.filter(
+    (el) => !(el[0] === "_" || el[0] === "@")
+  )
+  console.log("Filtered:", filtered)
```
Comment (sebastian-aranda) [inline src/js/components/TableApp.js:158]: This `console.log` should be removed.

**Review by sebastian-aranda — CHANGES_REQUESTED** [review]
Looks good @ugyballoons , I left a couple minor comments.

**Comment by ugyballoons** [comment]
Thanks Sebastian! Good spots as always :)

**Review by sebastian-aranda — APPROVED** [review]
Looks good, thanks for working on the comments!

**Pull request merged**